src/main/java/com/powsybl/openloadflow/network/impl/Transformers.java

```diff
@@ -274,9 +274,9 @@ public static int findTapPosition(PhaseTapChanger ptc, double angle) {
     /**
      * Find the tap position of a ratio tap changer corresponding to a given rho shift.
      */
-    public static int findTapPosition(RatioTapChanger rtc, double rho) {
+    public static int findTapPosition(RatioTapChanger rtc, double ptcRho, double rho) {
```

Comment:
Not clear to me why you add ptc tho to search a the rtc tap

Reply:
Because you use it to build the `PiModelArray`.